pkg/oc/cli/rollout/retry.go

```diff
@@ -35,11 +35,11 @@ type RetryOptions struct {
 	Builder           func() *resource.Builder
 	Mapper            meta.RESTMapper
 	Encoder           runtime.Encoder
-	Clientset         kclientset.Interface
+	Clientset         kexternalclientset.Interface
```

Comment:
usually I call this "kubernetes.Interface" and drop the import alias.... the "external" client is the new "default", so I don't see a point to name it "external"... (on the other hand, I feel a strong need to name internal clients "internal" ;-)

Reply:
Honestly, if possible narrow the client to core, only.